docs/_components/buttons.md

```diff
@@ -63,7 +65,7 @@ Use these button styles unless doing x and y, in which case, use [other buttons]
 
 # Other Buttons
 
-These buttons should be used when x-ing and y-ing.
+These buttons if the actions are not among the top two actions in a view, use an other button to convey action.
```

Comment:
"Use these buttons" or "These buttons should be used"

Reply:
ah good catch, updated.